llm_bench/python/llm_bench_utils/metrics_print.py

```diff
@@ -33,8 +31,8 @@ def print_metrics(
         output_str = ' '.join(['[{}]'.format(iter_str), output_str])
         log.info(output_str)
     if tms is not None:
-        iter_data['first_token_latency'] = tms[0] * 1000 if len(tms) > 0 else -1
-        iter_data['other_tokens_avg_latency'] = sum(tms[1:]) / (len(tms) - 1) * 1000 if len(tms) > 1 else -1
+        iter_data['first_token_latency'] = tms[0] * 1000 * batch_size if len(tms) > 0 else -1
```

Comment:
@QuPengfei  No need to multiply by batch_size, latency is calculated based on batch.

Reply:
here is one output example with the following command, the throughput is *2404.82*, that does not make sense.
===
OMP_NUM_THREADS=48 numactl  -m 0 -C 0-47  python benchmark.py -m /app/savedmodels/meta-llama/Llama-2-7b-chat-hf/pytorch/dldt/FP16 -d CPU -n 3 -p "It's done, and submitted. You can play 'Survival of the Tastiest' on Android, and on the web. Playing on the web works, but you have to simulate multiple touch for table moving and that can be a bit confusing. There is a lot I'd like to talk about. I will go through every topic, insted of making the typical what went right/wrong list. Concept Working over the theme was probably one of the hardest tasks which I had to face. Originally, I had an idea of what kind of game I wanted to develop, gameplay wise" -r /app/output/llama-2-7b-chat-FP16-16-128-128.1.csv -ic 128 -mc 2 -bs 16 --torch_compile_backend openvino --fuse_decoding_strategy -od /app/output --genai 

===
[ INFO ] [3] Input token size: 2048, Output size: 2064, Infer count: 128, Tokenization Time: 0.84ms, Detokenization Time: 0.82ms, Generation Time: 16.41s, Latency: 127.19 ms/16tokens
[ INFO ] [3] First token latency: 180.66 ms/16tokens, other tokens latency: 6.65 ms/16tokens, len of tokens: 128 * 16
[ WARNING ] [3] No hook data output for first infer latency and other infers latency
[ INFO ] [3] Max rss memory cost: 29424.91MBytes, max uss memory cost: 16365.31MBytes, max shared memory cost: 13059.60MBytes
[ INFO ] [3] Result MD5:['9d2f94ad15e3bb135cce357c2749bdbd', 'cd725121558552de59335411efc45bb7', 'cd725121558552de59335411efc45bb7', 'cd725121558552de59335411efc45bb7', '8ddb281023072f00cb141cfc43ada156', 'cd725121558552de59335411efc45bb7', '8ddb281023072f00cb141cfc43ada156', '0f0ff10bfa6fa71bd864d7eb873e67fe', 'cd725121558552de59335411efc45bb7', 'cd725121558552de59335411efc45bb7', '9d2f94ad15e3bb135cce357c2749bdbd', '45526431c81d4e0e742f2ee965f3c229', 'f3198aec72ac6dcf000ead8506c92606', 'cd725121558552de59335411efc45bb7', '842c7076cab6fe803b91c1108826ba7a', '8ddb281023072f00cb141cfc43ada156']
[ INFO ] <<< Warm-up iteration is excluded. >>>
[ INFO ] [Total] Iterations: 3
[ INFO ] [Average] Prompt[0] Input token size: 2048, 1st token lantency: 180.64 ms/16tokens, 2nd tokens latency: 6.65 ms/16tokens, 2nd tokens throughput: 2404.82 tokens/s
===

Reply:
2404.82 tokens/s is too big for 48 cores and f16.  that is not expected.

Reply:
@QuPengfei What is the number for -bs 1?

Reply:
@QuPengfei  This issue is specific to "--genai" option, a new PR will be created to fix this.

Reply:
@QuPengfei After this PR, the issue should be fixed now. https://github.com/openvinotoolkit/openvino.genai/pull/1244